includes/misc-functions.php

```diff
@@ -847,58 +847,64 @@ function array_column( $input = null, $columnKey = null, $indexKey = null ) {
  *
  * @since 1.3.2
  *
- * @param string $payment_key
+ * @param int $donation_id Donation ID.
  *
  * @return bool Whether the receipt is visible or not.
  */
-function give_can_view_receipt( $payment_key = '' ) {
+function give_can_view_receipt( $donation_id ) {
```

Comment:
@mehul0810 Add backward compatibility for `payment_key` as we discussed

Reply:
@ravinderk Added backward compatibility as discussed for the function `give_can_view_receipt`, in case if called directly.